python/sglang/srt/layers/attention/flashinfer_backend.py

```diff
@@ -45,13 +46,19 @@ def __init__(self, model_runner: ModelRunner):
         super().__init__()
 
         # Parse constants
-        if not _grouped_size_compiled_for_decode_kernels(
-            model_runner.model_config.num_attention_heads // model_runner.tp_size,
-            model_runner.model_config.get_num_kv_heads(model_runner.tp_size),
-        ):
-            self.decode_use_tensor_cores = True
+        if "SGLANG_FLASHINFER_USE_TENSOR_CORE" in os.environ:
+            self.decode_use_tensor_cores = (
+                os.environ["SGLANG_FLASHINFER_USE_TENSOR_CORE"].lower() == "true"
+            )
         else:
-            self.decode_use_tensor_cores = False
+            if not _grouped_size_compiled_for_decode_kernels(
```

Comment:
May we remove this `_grouped_size_compiled_for_decode_kernels` I think it's useless in FlashInfer v0.2 cc @yzh119

Reply:
Yes we can use some heuristic:
1. For fp16, `use_tensor_cores=True` when gqa_group_size > 4
2. For fp8, we can always enable `use_tensor_cores=True`

Reply:
I agree.